content/learn/command-line/node-repl.md

```diff
@@ -148,3 +148,5 @@ local.on('exit', () => {
   process.exit();
 });
 ```
+
+You can read more about the REPL module in the [repl documentation](/api/v19/repl).
```

Comment:
Just `/api/repl/` (add a trailing slash) should work.

Reply:
```suggestion
You can read more about the REPL module in the [repl documentation](/api/repl/).
```

Reply:
I tried doing that, it creates a link to `/en/api/repl` which is a 404 page

Reply:
My suggestion was /api/repl/ not /api/repl.

The links get the language code prefixed automatically. But actually all the links here have trailing slashes.